src/create_atoms.cpp

```diff
@@ -581,20 +585,11 @@ void CreateAtoms::command(int narg, char **arg)
   // print status
 
   MPI_Barrier(world);
-  double time2 = MPI_Wtime();
-
-  if (me == 0) {
-    if (screen) {
-      fprintf(screen,"Created " BIGINT_FORMAT " atoms\n",
-              atom->natoms-natoms_previous);
-      fprintf(screen,"  create_atoms CPU = %g secs\n",time2-time1);
-    }
-    if (logfile) {
-      fprintf(logfile,"Created " BIGINT_FORMAT " atoms\n",
-              atom->natoms-natoms_previous);
-      fprintf(logfile,"  create_atoms CPU = %g secs\n",time2-time1);
-    }
-  }
+  if (me == 0)
+    utils::logmesg(lmp, fmt::format("Created {} atoms\n"
+                        "  create_atoms CPU = {:<.3g} seconds\n",
```

Comment:
the '<' is not really needed. it was left over from some experiments. it requests left-justified (which is default); '>' will request right justified output. the ':' is required to indicate that the default setting is modified. the '.3' restricts to 3 up to 3 digits after the decimal point and 'g' the same style as '%g'. 
i've set the '.3g' since by default using '{}' will output with 15 digits precision, which looks very confusing to me for this kind of output.